tools/sof-kernel-log-check.sh

```diff
@@ -375,6 +375,9 @@ ignore_str="$ignore_str"'|elan_i2c i2c-ELAN0000:.*: failed to read current power
 # iwlwifi 0000:00:14.3: No beacon heard and the time event is over already...
 ignore_str="$ignore_str"'|iwlwifi [[:digit:].:]+: '
 
+# This can happen when starting snapd at boot time or when re-installing it.
+# https://github.com/thesofproject/sof-test/issues/874
+ignore_str="$ignore_str"'|I/O error, dev loop0, sector 0 op 0x0:.READ. flags 0x80700'
 
```

Comment:
@marc-hb  we also met error message: `kernel: I/O error, dev loop5, sector 0 op 0x0:(READ) flags 0x80700` , on TGLU_SKU0A32_SDCA and TGLH_SKU0A70_HDA, not only loop0.
http://sof-ci.sh.intel.com/#/result/planresultdetail/10879?model=TGLH_SKU0A70_HDA&testcase=verify-kernel-boot-log
http://sof-ci.sh.intel.com/#/result/planresultdetail/10879?model=TGLU_SKU0A32_SDCA&testcase=verify-kernel-boot-log

Reply:
Thanks good catch! Updated.

I found why: these systems have snap packages actually installed:

```
root@jf-tglu-sku0a32-sdca-02:~# losetup -a
/dev/loop1: [66306]:22289896 (/var/lib/snapd/snaps/gnome-3-34-1804_24.snap)
/dev/loop4: [66306]:22289898 (/var/lib/snapd/snaps/snap-store_433.snap)
/dev/loop2: [66306]:22289899 (/var/lib/snapd/snaps/snapd_7264.snap)
/dev/loop0: [66306]:22289895 (/var/lib/snapd/snaps/core18_1705.snap)
/dev/loop3: [66306]:22289897 (/var/lib/snapd/snaps/gtk-common-themes_1506.snap)
```

So the I/O error happens with a specific package? Strange.

snap consistency = same, zero snap packages installed on every device. Yet another Ansible job @greg-intel